### PR TITLE
IGNITE-12492 TDE - Phase-2. Documentation.

### PR DESCRIPTION
**Pull request by NSAmelchev**
Add master key rotation docs.

### CODE REVIEW

File: docs/_docs/security/master-key-rotation.adoc [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+|Method | Description
+|getMasterKeyName() | Gets the current master key name.
+|changeMasterKey(String masterKeyName) | Starts master key change process.
+|===
+
+=== From code
+
+The master key change process can be managed programmatically:
+
+[source,java]
```
Comment (artem-budnikov) [inline docs/_docs/security/master-key-rotation.adoc:59]: Please move the code example to a compilable java file (code-snippets/java/src/main/java/org/apache/ignite/snippets/TDE.java) and include snippets from that file here

File: docs/_docs/security/master-key-rotation.adoc [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+# Change the master key.
+control.sh|bat --encryption change_master_key newMasterKeyName
+----
+
+=== JMX
+
+You can also manage the master key change process via the `EncryptionMXBean` interface:
+
+[{table_opts}]
```
Comment (artem-budnikov) [inline docs/_docs/security/master-key-rotation.adoc:48]: The `table_opts` variable is not defined anywhere in the file. You should use explicit table configuration in this case, like so: 
[cols="1,3", opts="header"]

File: docs/_docs/security/master-key-rotation.adoc (hunk 1 above)
Comment (NSAmelchev) [reply docs/_docs/security/master-key-rotation.adoc]: Moved.

File: docs/_docs/security/master-key-rotation.adoc (hunk 2 above)
Comment (NSAmelchev) [reply docs/_docs/security/master-key-rotation.adoc]: Fixed. Thanks

**Pull request merged**